sympy/physics/mechanics/functions.py

```diff
@@ -422,6 +447,10 @@ def find_dynamicsymbols(expression, exclude=None):
             raise TypeError("exclude kwarg must be iterable")
     else:
         exclude_set = set()
+    if isinstance(expression, Vector) and reference_frame is None :
+            raise TypeError("You must provide a reference frame when passing in a vector expression.")
+    if isinstance(expression, Vector) :
+        expression=expression.to_matrix(reference_frame)
```

Comment:
add a space here `expression = expression.to_matrix(reference_frame)`

Reply:
I don't understand why the build is not succeeding. It is showing the following error:

File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.1.2.dev0-py2.7.egg/sympy/combinatorics/coset_table.py", line 910, in sympy.combinatorics.coset_table.coset_enumeration_r

Failed example:

    index

Expected:

    500

Got:

    1 

Although I haven't even touched these files.

Reply:
It seems that the coset table algorithm has some random methods that may sometimes lead to problems. That has nothing to do with your PR. The failing job will most probably pass when restarted.